bazel/BUILD

```diff
@@ -318,6 +318,11 @@ selects.config_setting_group(
     ],
 )
 
+config_setting(
+    name = "disable_admin_html",
+    values = {"define": "admin_html=disabled"},
+)
+
```

Comment:
Can you document this here? https://github.com/envoyproxy/envoy/tree/main/bazel#disabling-optional-features

Probably also worth a release note?